examples/ffd/src/intent_handler/intent_handler.c

```diff
@@ -74,6 +77,11 @@ static void proc_keyword_res(void *args) {
 #endif
 #if appconfAUDIO_PLAYBACK_ENABLED
         audio_response_play(id);
+#endif
+#if appconfLOW_POWER_ENABLED
+        if (inference_engine_keyword_queue_count() == 0) {
+            inference_engine_keyword_queue_complete();
+        }
```

Comment:
This bit of logic might need some protections against a race condition. If there are any thoughts on this or alternative ways to handle the indication that all keyword processing is done, I am open for suggestions.